Foundation/TSC/index.html

```diff
@@ -128,6 +128,12 @@
 %}
 
 <ul class="mt-6 space-y-2">
+{% include tsc-notes-li.html date="2025-11-13"
+   vote="to endorse June Andronick as LF representative for ETSI." %}
+{% include tsc-notes-li.html date="2025-10-30"
+   vote="to invite Indan Zupancic and Ivan Velickovic to the TSC." %}
+{% include tsc-notes-li.html date="2024-07-31"
```

Comment:
This was 2025 not 2024 (to be fixed in PDF as well)

Reply:
Indeed, my bad, will update